recipe/meta.yaml

```diff
@@ -5,23 +5,22 @@ package:
   version: {{ version }}
 
 source:
-  fn: PyScaffold-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/p/pyscaffold/PyScaffold-{{ version }}.tar.gz
   sha256: 8cc9dcbe2e4be48a0c25d5484ad0eea62ee0d490f82c44a5f7ca83bb8a2ca62f
 
 build:
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: python -m pip install --no-deps .
```

Comment:
I think you don't need this change. Only the change in Line 17

Reply:
This is borrowed from `ipython-feedstock` I had bad experience with some entrypoints when `pip` was not used. I'll take a look at the source if the entry point change and fix `17` later, but #9 worked without any modification :confused: